src/apps/insights/utils/supabaseAdapter.ts

```
@@ -0,0 +1,121 @@
+/**
+ * Mock Supabase client that translates calls to Firebase API
+ */
+export const createSupabaseAdapter = () => {
+  return {
+    from: (table: string) => ({
+      select: (columns = '*') => ({
+        order: (column: string, options?: { ascending?: boolean }) => ({
+          eq: (filterColumn: string, value: any) => ({
+            // This is a simplified adapter - for now just return a promise
+            // that calls the Firebase API
+            then: async (onResolve?: any, onReject?: any) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)